spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/SpatialConvolution.scala

```diff
@@ -597,14 +772,51 @@ object SpatialConvolution {
     initGradWeight: Tensor[Float] = null,
     initGradBias: Tensor[Float] = null,
     withBias: Boolean = true,
-    format: DataFormat = DataFormat.NCHW): SpatialConvolution = {
+    format: DataFormat = DataFormat.NCHW,
+    dilationW: Int = 1,
+    dilationH: Int = 1): SpatialConvolution = {
     new SpatialConvolution(nInputPlane, nOutputPlane, kW, kH, dW,
       dH, padW, padH, nGroup, propagateBack, wRegularizer, bRegularizer,
-      initWeight, initBias, initGradWeight, initGradBias, withBias, format)
+      initWeight, initBias, initGradWeight,
+      initGradBias, withBias, format, dilationW, dilationH)
   }
 }
 
 object Scale {
```

Comment:
private ?

Reply:
The **Scale** object does not belong to this PR, it is an existing code. 
Will make it to private and move it to a proper place in a separate PR.